src/main/java/uk/gov/hmcts/reform/et/syaapi/helper/JurisdictionCodesMapper.java

```diff
@@ -0,0 +1,92 @@
+package uk.gov.hmcts.reform.et.syaapi.helper;
+
+import org.springframework.stereotype.Service;
+import org.springframework.util.CollectionUtils;
+import uk.gov.hmcts.et.common.model.ccd.Et1CaseData;
+import uk.gov.hmcts.et.common.model.ccd.items.JurCodesTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.types.JurCodesType;
+import uk.gov.hmcts.reform.et.syaapi.constants.ClaimTypesConstants;
+import uk.gov.hmcts.reform.et.syaapi.constants.JurisdictionCodesConstants;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * Maps types of claims values {@link ClaimTypesConstants} to jurisdiction codes {@link JurisdictionCodesConstants}.
+ */
+@Service
+public class JurisdictionCodesMapper {
+    private static final Map<String, String> JURISDICTION_CODES = Map.ofEntries(
+        Map.entry(ClaimTypesConstants.BREACH_OF_CONTRACT, JurisdictionCodesConstants.BOC),
+        Map.entry(ClaimTypesConstants.UNFAIR_DISMISSAL, JurisdictionCodesConstants.UDL),
+        Map.entry(ClaimTypesConstants.WHISTLE_BLOWING, JurisdictionCodesConstants.PID),
+        Map.entry(ClaimTypesConstants.AGE, JurisdictionCodesConstants.DAG),
+        Map.entry(ClaimTypesConstants.DISABILITY, JurisdictionCodesConstants.DDA),
+        Map.entry(ClaimTypesConstants.ETHNICITY, JurisdictionCodesConstants.RRD),
+        Map.entry(ClaimTypesConstants.GENDER_REASSIGNMENT, JurisdictionCodesConstants.SXD),
+        Map.entry(ClaimTypesConstants.MARRIAGE_OR_CIVIL_PARTNERSHIP, JurisdictionCodesConstants.SXD),
+        Map.entry(ClaimTypesConstants.PREGNANCY_OR_MATERNITY, JurisdictionCodesConstants.MAT),
+        Map.entry(ClaimTypesConstants.RACE, JurisdictionCodesConstants.RRD),
+        Map.entry(ClaimTypesConstants.RELIGION_OR_BELIEF, JurisdictionCodesConstants.DRB),
+        Map.entry(ClaimTypesConstants.SEX, JurisdictionCodesConstants.SXD),
+        Map.entry(ClaimTypesConstants.SEXUAL_ORIENTATION, JurisdictionCodesConstants.DSO),
+        Map.entry(ClaimTypesConstants.ARREARS, JurisdictionCodesConstants.WA),
+        Map.entry(ClaimTypesConstants.HOLIDAY_PAY, JurisdictionCodesConstants.WTR_AL),
+        Map.entry(ClaimTypesConstants.NOTICE_PAY, JurisdictionCodesConstants.BOC),
+        Map.entry(ClaimTypesConstants.REDUNDANCY_PAY, JurisdictionCodesConstants.RPT)
+    );
```

Comment:
Could we not just do Map.of(ClaimTypesConstants.BREACH_OF_CONTRACT, JurisdictionCodesConstants.BOC, ClaimTypesConstants.UNFAIR_DISMISSAL, JurisdictionCodesConstants.UDL, ...) rather than Map.ofEntries(?

Reply:
Nope, it can accept only 10 values